apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -83,11 +79,11 @@ as described below
 ## Initializing Logging
 
 Logging can be initialized using `ConfigureServices` when
-constructing a `WebHostBuilder()`:
+constructing an `IWebHostBuilder`:
 
 {{sample:Logging.RegisterGoogleLogger2}}
 
-Or using `ConfigureLogging` when constructing a `WebHostBuilder()`:
+Or using `ConfigureLogging` when constructing an `IWebHostBuilder`:
```

Comment:
I'd expect this to be different from the type above, in a sort of "If you're using type X, do this; if you're using type Y, do that."